modules/text_server_adv/thorvg_svg_in_ot.cpp

```diff
@@ -97,7 +97,7 @@ FT_Error tvg_svg_in_ot_preset_slot(FT_GlyphSlot p_slot, FT_Bool p_cache, FT_Poin
 			if (parser->has_attribute("id")) {
 				const String &gl_name = parser->get_named_attribute_value("id");
 				if (gl_name.begins_with("glyph")) {
-					int dot_pos = gl_name.find(".");
+					int dot_pos = gl_name.find_char('.');
```

Comment:
Missed that you made these changes, will make a PR fixing this, these are not valid because this is an unexposed method, see:
* #99328

Reply:
Oops, sorry and thanks!

Reply:
Should have noticed as I did my own sweep with these before so it's on me!